### PR TITLE
Dropdown component- Adding placement prop for the error tooltip

### PR DESCRIPTION
**Pull request by barboaron**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6919/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6919

File: packages/wix-ui-tpa/src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -74,6 +77,7 @@ export class Dropdown extends React.Component<DropdownProps, State> {
     placement: 'bottom',
     mobileNativeSelect: false,
     upgrade: false,
+    errorTooltipPlacement: 'top-end',
```
Comment (jonathanadler) [inline packages/wix-ui-tpa/src/components/Dropdown/Dropdown.tsx:80]: I would move this to the `DropdownError` component

File: packages/wix-ui-tpa/src/components/Dropdown/DropdownError.tsx [hunk 2]
```diff
@@ -5,17 +5,19 @@ import { Tooltip } from '../Tooltip';
...
+  placement?: Placement;
 }
 
 export const DropdownError: React.FC<DropdownErrorProps> = (props) => {
   return (
     <Tooltip
       className={st(classes.root, props.className)}
       data-hook={DATA_HOOKS.errorTooltip}
-      placement="top-end"
+      placement={props.placement || 'top-end'}
```
Comment (jonathanadler) [inline packages/wix-ui-tpa/src/components/Dropdown/DropdownError.tsx:20]: I would move the default prop value to this file, and then you could just write
```suggestion
      placement={props.placement}
```

**Review by jonathanadler — COMMENTED** [review]
Looks great, just left some minor comments

File: packages/wix-ui-tpa/src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (barboaron) [reply packages/wix-ui-tpa/src/components/Dropdown/Dropdown.tsx]: done

File: packages/wix-ui-tpa/src/components/Dropdown/DropdownError.tsx (hunk 2 above)
Comment (barboaron) [reply packages/wix-ui-tpa/src/components/Dropdown/DropdownError.tsx]: done

**Review by jonathanadler — APPROVED** [review]
Nice!

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6919/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6919

**Pull request merged**